core/src/test/java/ai/timefold/solver/core/impl/domain/solution/cloner/AbstractSolutionClonerTest.java

```diff
@@ -1050,51 +1060,51 @@ private void assertDeepCloningEntityClone(TestdataFieldAnnotatedDeepCloningEntit
-        Map<String, String> cloneShadowVariableMap = cloneEntity.getShadowVariableMap();
+        var originalShadowVariableMap = originalEntity.getShadowVariableMap();
+        var cloneShadowVariableMap = cloneEntity.getShadowVariableMap();
         if (originalShadowVariableMap == null) {
             assertThat(cloneShadowVariableMap).isNull();
         } else {
             assertThat(cloneShadowVariableMap).isNotSameAs(originalShadowVariableMap);
             assertThat(cloneShadowVariableMap).hasSameSizeAs(originalShadowVariableMap);
-            for (String key : originalShadowVariableMap.keySet()) {
+            for (var key : originalShadowVariableMap.keySet()) {
                 assertThat(cloneShadowVariableMap.get(key)).isSameAs(originalShadowVariableMap.get(key));
             }
         }
 
-        Map<List<String>, String> originalStringListToStringMap = originalEntity.getStringListToStringMap();
-        Map<List<String>, String> cloneStringListToStringMap = cloneEntity.getStringListToStringMap();
+        var originalStringListToStringMap = originalEntity.getStringListToStringMap();
+        var cloneStringListToStringMap = cloneEntity.getStringListToStringMap();
         if (originalStringListToStringMap == null) {
             assertThat(cloneStringListToStringMap).isNull();
         } else {
             assertThat(cloneStringListToStringMap).isNotSameAs(originalStringListToStringMap);
             assertThat(cloneStringListToStringMap).hasSameSizeAs(originalStringListToStringMap);
-            for (List<String> key : originalStringListToStringMap.keySet()) {
+            for (var key : originalStringListToStringMap.keySet()) {
                 assertThat(cloneStringListToStringMap.get(key)).isSameAs(originalStringListToStringMap.get(key));
             }
         }
 
-        Map<String, List<String>> originalStringToStringListMap = originalEntity.getStringToStringListMap();
-        Map<String, List<String>> cloneStringToStringListMap = cloneEntity.getStringToStringListMap();
+        var originalStringToStringListMap = originalEntity.getStringToStringListMap();
+        var cloneStringToStringListMap = cloneEntity.getStringToStringListMap();
         if (originalStringToStringListMap == null) {
             assertThat(cloneStringToStringListMap).isNull();
         } else {
             assertThat(cloneStringToStringListMap).isNotSameAs(originalStringToStringListMap);
             assertThat(cloneStringToStringListMap).hasSameSizeAs(originalStringToStringListMap);
-            for (String key : originalStringToStringListMap.keySet()) {
-                assertThat(cloneStringToStringListMap.get(key)).isEqualTo(originalStringToStringListMap.get(key));
+            for (var key : originalStringToStringListMap.keySet()) {
+                assertThat(cloneStringToStringListMap).containsEntry(key, originalStringToStringListMap.get(key));
```

Comment:
Especially if the PR has changes like this, the review goes from "check imports" to "check 800 files for any possible problem in the test".

Reply:
Okay. Should I revert it or adhere to the proposed guidelines in the next related changes?

Reply:
Don't revert anymore. I already reviewed everything.